sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -614,7 +628,10 @@ public virtual async Task StartProcessingAsync(CancellationToken cancellationTok
         ///
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> instance to signal the request to cancel the stop operation.  If the operation is successfully canceled, the <see cref="EventProcessorClient" /> will keep running.</param>
         ///
-        public virtual async Task StopProcessingAsync(CancellationToken cancellationToken = default)
+        public virtual void StopProcessing(CancellationToken cancellationToken = default)
+            => StopProcessingImplAsync(false, cancellationToken).EnsureCompleted();
+
+        private async Task StopProcessingImplAsync(bool async, CancellationToken cancellationToken)
```

Comment:
Please rename to `StopProcessingInternalAsync` and add doc comments.

Reply:
Fixed